userguide/01. Intro to Channels.ipynb

```diff
@@ -0,0 +1,586 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "from __future__ import absolute_import\n",
+    "from __future__ import print_function\n",
+    "\n",
+    "from tornado import gen\n",
+    "from flowz import app\n",
+    "from flowz.channels import *"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Introduction to Channels"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Channel"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A `Channel` gives a means of asynchronous iteration over items coming from some upstream source.\n",
+    "\n",
+    "A consumer of a `Channel` uses its `next()` method to iteratively receive items as the channel makes them available; when the channel is exhausted, subsequent invocations of `next()` result in a `ChannelDone` exception.  Those calls to `next()`, however, would need to be made in the context of a tornado coroutine, which gets complex pretty quickly.\n",
+    "\n",
```

Comment:
You may want to highlight the `tee` functionality right here in the intro, since that's once of the most important design considerations and one of the arguments for having our own "channel" abstraction rather than looking to python 3 async iterators.